src/components/AnswerCard/index.js

```diff
@@ -0,0 +1,131 @@
+  const [answering, setAnswering] = useState(false);
+
+  const comment = {
+    user: "Jorginho Heizenhower",
+    text: "Você poderia ter apenas aconselhado reinstalar o KDE. :)"
+  };
+
+  const comments = [comment, comment];
+
+  const handleUpvote = () => {
+    setDownvote(false);
+    setUpvote(!upvote);
+  };
+
+  const handleDownvote = () => {
+    setUpvote(false);
+    setDownvote(!downvote);
+  };
+
+  const handleAnswering = () => {
+    setAnswering(!answering);
+  }
+
+  return (
+    <>
+      <Box
+        sx={{
+          width: "100%",
+          background: "#393E41",
+          border: "1px solid #F0F0F0",
+          padding: "1.5em",
+          borderRadius: "10px",
+          display: "flex",
+          flexDirection: "column",
+          gap: "1em",
+          borderLeft: "7px solid #1976D2",
+          "&.accepted": { "border-left": "7px solid #49BE25" },
+          "&.normal": { "border-left": "7px solid #1976D2" }
+        }}
+        className={accepted ? "accepted" : "normal"}
+      >
+        <CardHeader
+          username={answer.user.name}
+          userpicture={answer.user.img}
+          time={answer.time}
+          resumed={resumed}
+        />
+        <CardContent
+          tldr={answer.tldr}
+        />
+        <Divider sx={{ borderColor: "#F0F0F0" }} />
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
+        >
+          <Box
+            sx={{
+              width: "100%",
+              display: "flex",
+              gap: "1em",
+            }}
+          >
+            <Button
+              label={answer.status.likes}
+              icon={<ArrowUpwardIcon color="white" fontSize="small" />}
+              background={"#49BE25"}
+              fontColor={"#181818"}
+              onClick={handleUpvote}
+              isActive={upvote}
+            />
+            <Button
+              label={answer.status.comments}
+              icon={<ArrowDownwardIcon color="white" fontSize="small" />}
+              background={"#E80000"}
+              fontColor={"#181818"}
+              onClick={handleDownvote}
+              isActive={downvote}
+            />
+          </Box>
+          <Button
+            label={"Responder"}
+            background={"#1976D2"}
+            fontColor={"#F0F0F0"}
+            onClick={handleAnswering}
+            isActive={answering}
+          />
+        </Box>
+        {answering && (
+          <TextField
+            sx={{
+              borderRadius: "5px",
+              color: "white",
+              background: "#F0F0F0",
+              fontSize: "16px"
+            }}
+            InputProps={{
+              endAdornment: (
+                <IconButton>
+                  <SendIcon sx={{ color: "#1976D2" }} fontSize="small" />
+                </IconButton>
+              )
+            }}
+            multiline
+          />
+        )}
+      </Box>
```

Comment:
Uma dúvida, como esse é o `AnswerCard` essa parte é referente a alguém comentando a resposta? Ou relacionado a adicionar uma resposta? Porque se for o primeiro caso acho que a nomenclatura deveria ser `Comentar` e não `Responder`.